config/initializers/money.rb

```diff
@@ -61,8 +61,8 @@
     :symbol_first        => false,
     :subunit             => "Kopek",
     :subunit_to_unit     => 100,
-    :thousands_separator => ".",
-    :decimal_mark        => ","
+    :thousands_separator => ",",
```

Comment:
@yunixon for russian locale usually thousand separator is space ` ` and decimal mark is comma `,`

Reply:
@yunixon And I think it's not a good idea to change it just to fix frontend